nautobot_golden_config/utilities/helper.py

```diff
@@ -151,15 +151,18 @@ def get_repository_working_dir(
     """
     match_rule = getattr(global_settings, f"{repo_type}_match_rule")
 
-    if not match_rule:
+    if not match_rule and repo_type == "backup":
         return global_settings.backup_repository.first().filesystem_path
```

Comment:
I think the only thing that needs to change is
```suggestion
        return getattr(global_settings, f"{repo_type}_repository).first().filesystem_path
```

Reply:
ok, let me try that.

Reply:
so, yeah, that's all that was needed.